### ISSUE TITLE
after update to stable 1.29.5 it still shows 1.29.4 when i do rvm --version 

### ISSUE THREAD

**Issue by mariuz** [rvm/rvm#4521]
after update to stable 1.29.5 it still shows 1.29.4 when i do rvm --version 

` rvm get stable
Downloading https://get.rvm.io
Downloading https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer.asc
Verifying /home/ubuntu/.rvm/archives/rvm-installer.asc
gpg: Signature made Sat Mar 31 21:47:44 2018 UTC
gpg:                using RSA key 62C9E5F4DA300D94AC36166BE206C29FBF04FF17
gpg: Good signature from "Michal Papis (RVM signing) <mpapis@gmail.com>" [unknown]
gpg:                 aka "Michal Papis <michal.papis@toptal.com>" [unknown]
gpg:                 aka "[jpeg image of size 5015]" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 409B 6B17 96C2 7546 2A17  0311 3804 BB82 D39D C0E3
     Subkey fingerprint: 62C9 E5F4 DA30 0D94 AC36  166B E206 C29F BF04 FF17
GPG verified '/home/ubuntu/.rvm/archives/rvm-installer'
Downloading https://github.com/rvm/rvm/archive/1.29.5.tar.gz
Downloading https://github.com/rvm/rvm/releases/download/1.29.5/1.29.5.tar.gz.asc
gpg: Signature made Wed Dec 12 11:25:22 2018 UTC
gpg:                using RSA key 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
gpg: Good signature from "Piotr Kuczynski <piotr.kuczynski@gmail.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 7D2B AF1C F37B 13E2 069D  6956 105B D0E7 3949 9BDB
GPG verified '/home/ubuntu/.rvm/archives/rvm-1.29.5.tgz'
Upgrading the RVM installation in /home/ubuntu/.rvm/
    RVM PATH line found in /home/ubuntu/.mkshrc /home/ubuntu/.profile /home/ubuntu/.bashrc /home/ubuntu/.zshrc.
    RVM sourcing line found in /home/ubuntu/.profile /home/ubuntu/.bash_profile /home/ubuntu/.zlogin.
Upgrade of RVM in /home/ubuntu/.rvm/ is complete.
  * No new notes to display.

rvm --version
rvm 1.29.4 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]

`

**Comment by pkuczynski** [comment]
Yeah, I forgot to update VERSION. Will do it with next release, today.

**Comment by gonace** [comment]
Hi
Are there any new/more problems with the version?

```
      01 Warning, new version of rvm available '1.29.7', you are using older version '1.29.6'.
      01 You can disable this warning with:    echo rvm_autoupdate_flag=0 >> ~/.rvmrc
      01 You can enable  auto-update  with:    echo rvm_autoupdate_flag=2 >> ~/.rvmrc
```

I'm getting this warning on multiple machines after upgrading to 1.29.6

**Comment by pkuczynski** [comment]
Apologies. That's my fault. I just fixed this and will be available in the next release (end of the year). You can in the meantime run `rvm get master` to get rid of it
